statler_site/views.py

```diff
@@ -9,7 +9,8 @@ def healthCheck(request):
 def getPlayListPage(request):
     """returns an html page which will use JavaScript
     to get and display data"""
-    return render_to_response('play-list-page.html')
+    # 'all'  is the hardcoded play list name for getting all plays
```

Comment:
The spec says that endpoint returns just a list of plays. But I will remove the hardcoded 'all' from the views for the other reasons you mentioned. There will still be a hardcoded 'all' in this location, because this is telling the play list page what url it should get JSON from.